oletools/olevba.py

```diff
@@ -496,9 +496,11 @@ def __init__(self, stream_path, variable, expected, value):
     # any MS Office application:
     'Runs when the file is opened (using InkPicture ActiveX object)':
         # ref:https://twitter.com/joe4security/status/770691099988025345
-        (r'\w+_Painted',),
+        (r'\w+_Painted|Painting',),
     'Runs when the file is opened and ActiveX objects trigger events':
         (r'\w+_(?:GotFocus|LostFocus|MouseHover)',),
+    'Runs when the file is opened and a Form frame layout is initialised':
+        (r'(?:Frame|MultiPage)1_Layout',),        
```

Comment:
I wasn't sure if this could erroneously capture objects that could be mis-classified as autoexec so I decided to restrict to Frame and Multipage.